application/GW2AddonManager/UI/OpeningPage/OpeningViewModel.cs

```diff
@@ -97,20 +96,20 @@ public string Log
         public Visibility LogVisibility => Log.Length > 0 ? Visibility.Visible : Visibility.Collapsed;
 
         [DependsOn("Addons")]
-        public bool CanInstallAny => CheckedAddons.Any(x => {
+        public bool CanInstallAny => _configurationProvider.UserConfig.GamePath != null && CheckedAddons.Any(x => {
```

Comment:
Why not use `GamePathMissing` here and below now that it exists?

Reply:
Hadn't added `GamePathMissing` as a property in `OpeningViewModel`, only `MainWindowViewModel`, but can add it and use that though.